lib/src/route_generator.dart

```diff
@@ -172,28 +172,47 @@ class RouteGenerator {
         nodes.forEach((node) {
           node.fileInfoList.forEach((info) {
             info.routes.forEach((route) {
-              routeNames.add(route.ffRoute.name.replaceAll("\"", ""));
+              routeNames.add(route.ffRoute.name.replaceAll('\"', ''));
               caseSb.write(route.caseString);
             });
           });
         });
       }
 
-      sb.write(rootFile.replaceAll("{0}", caseSb.toString()));
+      sb.write(rootFile.replaceAll('{0}', caseSb.toString()));
 
       if (generateRouteNames) {
-        sb.write("\n");
-        sb.write("List<String> routeNames = ${json.encode(routeNames)};");
+        sb.write('\n');
+        sb.write('List<String> routeNames = ${json.encode(routeNames)};');
+      }
+
+      if (generateRouteConstants) {
```

Comment:
it's better to add annotation (description and parameter) for every route const.